drivers/user_permission_report/app/controllers/user_permission_report/warehouse_reports/reports_controller.rb

```diff
@@ -0,0 +1,30 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module UserPermissionReport::WarehouseReports
+  class ReportsController < ApplicationController
+    include WarehouseReportAuthorization
+    include ArelHelper
+
+    before_action :set_limited, only: [:index]
+    before_action :set_group_associations
+
+    def index
+      @users = User.active.order(:last_name).includes(:roles, access_groups: @group_associations)
```

Comment:
This should be paginated, maybe 25 users per page.
I _think_ we'll also need a simple search by name/email.